src/helpViewer/helpPreviewer.ts

```diff
@@ -231,22 +228,41 @@ export class RLocalHelpPreviewer {
             return undefined;
         }
 
+        const rd2HtmlArgs = [
```

Comment:
For such use case, it might be worth having a dedicated script since it is not an one-liner.